packages/cmf/src/cmfConnect.js

```diff
@@ -176,5 +185,7 @@ export default function cmfConnect({
 			mergeProps,
 			{ ...rest },
 		)(hoistStatics(CMFContainer, WrappedComponent));
+		Connected.CMFContainer = CMFContainer;
```

Comment:
To being able to tests this class ... it's a bit like a singleton you can't access because there is no link from outside.